pkg/operator/ceph/csi/version.go

```diff
@@ -25,22 +25,18 @@ import (
 )
 
 var (
-	//minimum supported version is 3.0.0
-	minimum = CephCSIVersion{3, 0, 0}
+	//minimum supported version is 3.3.0
+	minimum = CephCSIVersion{3, 3, 0}
 	//supportedCSIVersions are versions that rook supports
-	releasev310          = CephCSIVersion{3, 1, 0}
-	releasev320          = CephCSIVersion{3, 2, 0}
 	releasev330          = CephCSIVersion{3, 3, 0}
 	releasev340          = CephCSIVersion{3, 4, 0}
 	supportedCSIVersions = []CephCSIVersion{
 		minimum,
-		releasev310,
-		releasev320,
 		releasev330,
 		releasev340,
 	}
-	// omap generator is supported in v3.2.0+
-	omapSupportedVersions = releasev320
+	// omap generator is supported in v3.3.0+
+	omapSupportedVersions = releasev330
```

Comment:
we don't need this anymore as we default to 3.3

Reply:
@Madhu-1 so we will only  get rid of the source code comment, isnt it ?

Reply:
we need to remove `omapSupportedVersions` variable and its checks

Reply:
ok. got rid of it.. @Madhu-1 ptal.. thanks